dbg_files.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Provie file paths to archive for debug image as relative to src subdir
+#
+SRC_DIR_LIST="\
```

Comment:
can you move this into each make file so that we can maintain the list in each individual makefile. that will be easier to maintain as if we add or delete in that makefile.

Reply:
ok. Makes sense.